Fix for Radio Button Group breaks when radio value is an integer Issue #364

bengfarrell opened the pull request:
https://github.com/adobe/spectrum-web-components/issues/364

I've tested this fix by extending the Radio-Group component in my project, and don't see anything bad. However, in this pull request, I'm not testing because I don't have the local environment set up

<!--- Provide a general summary of your changes in the Title above -->

## Description

<!--- Describe your changes in detail -->
Simply wrapped the offending selector in quotes.

## Related Issue

<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->
Radio Button Group breaks when radio value is an integer Issue #364
https://github.com/adobe/spectrum-web-components/issues/364

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->
Radio Button Groups could not contain radio buttons with a value of type integer previously. The Radio Group, when set selected would throw an error due to query selection

## How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

Have not tested this fix at all, other than extending Radio Button Group on my project to patch this issue temporarily

## Screenshots (if appropriate):

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Westbrook commented:
@bengfarrell Thanks for putting this together! If you could just do one more thing here, I look forward to approving this!

If you could update the code found https://github.com/adobe/spectrum-web-components/blob/master/packages/radio-group/test/radio-group.test.ts#L55-L59 to include number based `value` and `selected` attributes, and then pair that with updates to https://github.com/adobe/spectrum-web-components/blob/master/packages/radio-group/test/radio-group.test.ts#L198 that match those values so that we are protected from future regressions in this area, I'd be much obliged.

bengfarrell commented:
OK, updated as per @Westbrook. As I state in my commit, unfortunately, I'd have to spend a fair bit of time figuring out why things aren't building for me, and why my tests seem to be running silently (not breaking when I intend them to), so I haven't validated my new tests. However, I added a new test with a new block of HTML. The new HTML simply includes radio buttons with integer values, and the group selects one of them. Without my change, this test would produce a runtime error -  so I assume it's good, but I am flying blind here
